### DISCUSSION TITLE
Unable to update Controlnet

### DISCUSSION THREAD

**Discussion by NoobArtist85** [Mikubill/sd-webui-controlnet#2018] [General]
I followed the below steps but controlnet wont update from v1.1.227
1. Go to stable-diffusion-webui\extensions
2. Rename sd-webui-controlnet folder to "!! Backup-2-sd-webui-controlnet"
3. open a command prompt from the above location and run the command "git clone https://github.com/Mikubill/sd-webui-controlnet.git"
4. ones the sd-webui-controlnet directory is created, copy the Models from the backup folder

**Comment by huchenlei** [comment]
Rename the folder will not make the extension disabled. You need to move the backup folder out of extensions folder.

**Reply by NoobArtist85** [reply]
Thank you huchenlei, this worked.
